src/project-tests/pomodoro-clock-tests.js

```diff
@@ -511,6 +516,37 @@ export default function createPomodoroClockTests() {
         });
       });
 
+      it(`27. The audio element with id="beep" must be 1 second or longer.`,
+      function() {
+        FCC_Global.assert.isAbove(
+          document.getElementById('beep').duration,
+          1,
+          `Audio element with id="beep" is not at least 1 second long.`
+        );
+      });
+
+      it(`28. The audio element with id of "beep" must stop playing and be
+      rewound to the beginning when the element with the id of "reset" is
+      clicked.`,
+      function() {
+        var audioElem = document.getElementById('beep');
+
+        audioElem.play();
+        resetTimer();
```

Comment:
Should there be a timeout to play the audio element? The tests are passing for me. Test 21 was using promises and timeouts for starting, pausing, and rewinding (I'd prefer saying reset, but this is fine).